client/js/libs/handlebars/index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+module.exports = {
+	colorClass: require("./colorClass"),
+	diff: require("./diff"),
+	equal: require("./equal"),
+	firstLetterUppercase: require("./firstLetterUppercase"),
+	localedate: require("./localedate"),
+	localetime: require("./localetime"),
+	modes: require("./modes"),
+	parse: require("./parse"),
+	roundBadgeNumber: require("./roundBadgeNumber"),
+	slugify: require("./slugify"),
+	ternary: require("./ternary"),
+	tojson: require("./tojson"),
+	tz: require("./tz"),
+	users: require("./users"),
+};
```

Comment:
This doesn't exist anymore because we don't use any helpers. So we will need to figure this out when we want to use any of the helpers, but for now it is not a problem.